pymongo/change_stream.py

```diff
@@ -16,35 +16,63 @@
 
 import copy
 
+from bson.son import SON
+
+from pymongo import common
+from pymongo.collation import validate_collation_or_none
+from pymongo.command_cursor import CommandCursor
 from pymongo.errors import (ConnectionFailure, CursorNotFound,
                             InvalidOperation, PyMongoError)
 
 
 class ChangeStream(object):
-    """A change stream cursor.
+    """The change stream cursor base class.
 
     Should not be called directly by application developers. Use
-    :meth:`~pymongo.collection.Collection.watch` instead.
+    helper methods :meth:`~pymongo.collection.Collection.watch`,
+    :meth:`~pymongo.database.Database.watch`, and
+    :meth:`~pymongo.mongo_client.MongoClient` instead.
 
     .. versionadded: 3.6
     .. mongodoc:: changeStreams
     """
-    def __init__(self, collection, pipeline, full_document,
+    def __init__(self, target, pipeline, full_document,
                  resume_after=None, max_await_time_ms=None, batch_size=None,
-                 collation=None, session=None):
-        self._collection = collection
+                 collation=None, start_at_operation_time=None, session=None):
```

Comment:
Since we always pass all of these arguments when creating a ChangeStream can you make all of them required?